service/src/models/env-config.ts

```diff
@@ -0,0 +1,8 @@
+export interface EnvConfig {
+    publicKey: string,
+    maxRessurectiomTime: number,
+    minBounty: number,
+    minDiggingFees: number,
```

Comment:
Some (maybe all) of the number fields will need to be ethers `BigNumber` once we introduce ethers, since they will be denominated in ETH.